psi4/src/read_options.cc

```diff
@@ -941,6 +941,8 @@ int read_options(const std::string &name, Options & options, bool suppress_print
     options.add_int("SAPT_FDDS_DISP_NUM_POINTS", 10);
     /*- Lambda shift in the space morphing for the FDDS Dispersion time integration !expert -*/
     options.add_double("SAPT_FDDS_DISP_LEG_LAMBDA", 0.3);
+    /*- Minima rho cutoff for the in the LDA response for FDDS -*/
+    options.add_double("SAPT_FDDS_V2_RHO_CUTOFF", 1.e-6);
```

Comment:
Agree with @loriab on the minimum/minima switcharoo, but should this be `!expert`?  I would think so.

Reply:
Good point, fixed.